app/assets/stylesheets/login.css.scss

```diff
@@ -0,0 +1,207 @@
+#login-bg {
```

Comment:
Since you are using scss might as well make use of variables. I see #fff appear everywhere and such.